src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -45,22 +50,33 @@ const Container = styled.span`
 `;
 
 const translateUpAndFadeIn = keyframes`
-    from { opacity: 0; transform: translateY(-14px); };
-    to { opacity: 1; transform: translateY(-28px); };
+    from { opacity: 0; transform: translateX(-50%) translateY(-14px); };
+    to { opacity: 1; transform: translateX(-50%) translateY(-28px); };
 `;
 
 const translateDownAndFadeOut = keyframes`
-    from { opacity: 1; transform: translateY(-28px); };
-    to { opacity: 0; transform: translateY(-14px); };
+    from { opacity: 1; transform: translateX(-50%) translateY(-28px); };
+    to { opacity: 0; transform: translateX(-50%) translateY(-14px); };
 `;
 
 const Toast = styled.div<{ isActive: boolean }>`
-  position: absolute;
-  background: black;
-  padding: 4px 8px;
+  position: fixed;
```

Comment:
there's a toast component in Toast.tsx that we might be able to use here!
you can check for usages of useToastActions for examples

Reply:
So much cleaner! but getting a weird duplication/jitter error.

https://user-images.githubusercontent.com/13339581/172883151-c77e7aa9-2a08-4be1-ad0c-9975ad9d9936.mov

I verified the [`useCallback` responsible for pushing the toast](https://github.com/gallery-so/gallery/pull/807/commits/010403c55fc9439521b37c5e175d2ec96b67a487#diff-9f844a7dbe8597affea197b397fc099857b0b7f6c41e51922234ff3020e094c5R18-R23) only runs once. So I wonder if this is an error with rerendering the toast based on ID. (Currently the key is just the toast message, which would be the same across collections)

I get this error in console:

> Warning: Encountered two children with the same key, `Copied link to clipboard`. Keys should be unique so that components maintain their identity across updates.

Reply:
Fixed!